packages/server/package.json

```diff
@@ -7,11 +7,10 @@
   "license": "UNLICENSED",
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "nest build",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "dev": "nest start --watch --preserveWatchOutput",
+    "build": "nest build --webpack --webpackPath webpack.config.js",
+    "dev": "nest build --webpack --webpackPath webpack-hmr.config.js",
```

Comment:
should this be `nest start` instead of build since we're merging into master?

Reply:
no, it's weird but the `dev` script works because webpack-hmr.config.js  makes it so the build process kicks off the server. However, without the hmr config, it just builds.

Reply:
interesting ok! that's fine then